examples/experimental/onnx/onnx_ptq_classification.py

```diff
@@ -12,30 +12,76 @@
 """
 
 import argparse
+import numpy as np
 
 from typing import List
+from typing import Optional
 
 import onnx
 
 from nncf.experimental.post_training.compression_builder import CompressionBuilder
 from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantization
 from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantizationParameters
-from nncf.experimental.onnx.dataloaders.imagenet_dataloader import create_dataloader_from_imagenet_torch_dataset
+from nncf.experimental.onnx.datasets.imagenet_dataset import create_imagenet_torch_dataset
+from nncf.experimental.post_training.api.metric import Metric
+from nncf.common.utils.logger import logger as nncf_logger
+
+
+class Accuracy(Metric):
+
+    """
+    The classification accuracy metric is defined as the number of correct predictions
+    divided by the total number of predictions.
+    It measures the proportion of examples for which the predicted label matches the single target label.
+    Metric is calculated as a percentage.
+    """
+
+    def __init__(self, top_k: int = 1):
+        super().__init__()
+        self._top_k = top_k
+        self._name = f'accuracy@top{self._top_k}'
+        self._matches = []
+
+    @property
+    def avg_value(self):
```

Comment:
Do you have tests on Metrics calulcation?

Reply:
Added test to `tests/experimental/post_training/test_api.py`